### ISSUE TITLE
Nonstackables Unstack in Shulker Boxes (overwrites items)

### ISSUE THREAD

**Issue by QuirkyKitKit** [Mikedeejay2/SimpleStackPlugin#76]
Nonstackables will overwrite themselves or other items in a Shulker Box. 

To replicate: 
Configure normally non-stackable items to stack
Place a stack of those inside of a Shulker Box (more than 27 or a combination of items totaling more than 27 slots in vanilla)
Pick up and then place the Shulker Box 
Open the Shulker Box

The nonstackables will be unstacked and some additional items will be overwritten. 

This does not happen in creative, only survival.

**Comment by Mikedeejay2** [comment]
Thanks for the bug report! I'm unable to reproduce this bug in my server but I am aware of other times that this occurred that have been patched. What version of Simple Stack are you running on? What Minecraft version are you running on? Are there any errors being thrown to the console? Thanks!

**Comment by QuirkyKitKit** [comment]
This may have been a server specific issue, I apologize for jumping to report it. 

The server runs Minecraft 1.16.5 and Spigot. Yesterday we removed all plugins then reuploaded them, this seemed to have somehow fixed the issue (we used the exact same files we had removed). Kind of confused but extremely happy it is now working. 

While the issue was happening, the console never reported any errors or warnings.

Thank you for your swift response and keep up the good work!!
